pinot-common/src/main/java/org/apache/pinot/sql/parsers/CalciteSqlParser.java

```diff
@@ -397,7 +403,8 @@ private static void rewriteNonAggregationGroupByToDistinct(PinotQuery pinotQuery
         pinotQuery.setGroupByList(Collections.emptyList());
       } else {
         selectIdentifiers.removeAll(groupByIdentifiers);
-        throw new SqlCompilationException(String.format("For non-aggregation group by query, all the identifiers in select clause should be in groupBys. Found identifier: %s",
+        throw new SqlCompilationException(String.format(
+            "For non-aggregation group by query, all the identifiers in select clause should be in groupBys. Found identifier: %s",
```

Comment:
@siddharthteotia, fyi, does this break any of your use cases for PQL -> SQL migration?

Reply:
This line is just auto-format